translations/mgcp.js

```diff
@@ -2117,10 +2117,13 @@ mgcp = {
             // Order is important:
             // First the MGCPv4 FCODES, then the common ones. This ensures that the common ones don't
             // stomp on the V4 ones
-            mgcp.rules.fcodeOne2oneV4.push.apply(mgcp.rules.fcodeOne2oneV4,fcodeCommon.one2one);
             mgcp.rules.fcodeOne2oneV4.push.apply(mgcp.rules.fcodeOne2oneV4,mgcp.rules.fcodeOne2oneOut);
+            mgcp.rules.fcodeOne2oneV4.push.apply(mgcp.rules.fcodeOne2oneV4,fcodeCommon.one2one);
```

Comment:
Seems this order switched and the comment says it's important so just want to double check it's intentional.

Reply:
The order is important and it is intentional. When I swapped to the foreach method I noticed that the loop wasn't doing what it was supposed to do. I added a rule to the export list list and found that it wasn't taking precedence over a common rule which made me look at the ordering.

The idea is to set up a set of one to one rules and drop the ones that clash. The rules are in an array and it is traversed in order. Later things get dropped if they clash.
E.g. Looking at FCodes:
* Add all of the MGCP specific FCode rules
  + FCodes that are not in the common list
* Add all of the MGCP Export only specific FCode rules
  + "building=yes" -> AL015
* Add all of the common FCode rules
  + "building=yes" -> AL013    <== This gets dropped since it clashes with an existing rule

To make things cleaner, I'm thinking about getting rid of the common list and pushing all of the schema specific FCode rules into the individual translation files. This would remove one "require" from the JS.